auth/service-to-service/auth.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Demonstrates how to send authenticated service-to-service requests, eg 
+for Cloud Run or Cloud Functions"""
+
+# [START run_service_to_service_auth]
```

Comment:
Cross-product reuse is poorly defined, but this should probably add the google_auth region tag used in the nodejs version I linked earlier.

If the plan is to switch the GCF sample to use this, we can add the functions region tag in as well and delete the current sample after docs update.

Wrapping multiple region tags is "first in, last out" order.

Reply:
Filed a follow-up bug.  It's not as simple as adding in the region tags here, b/c the sample is expecting it to be a http Function.  This [doc](https://cloud.google.com/functions/docs/securing/authenticating#functions-bearer-token-example-python) would have to be updated.